src/components/SingleReview.jsx

```diff
@@ -14,6 +18,21 @@ export const SingleReview = () => {
             setReview(review);
         })
     }, [review_id])
+
+    const Vote = ({review_id},  vote) => {
+        setReview((currReview)=> {
+            const updatedReview = {...currReview}
+            updatedReview.votes += vote;
+            return  updatedReview; }) 
+
+            patchVotesReview(review_id, vote).then((res) => 
+                res !== 200 ? setReview((currReview)=> {
```

Comment:
Awesome check to ensure that the request was successful and if not to revert the change